### PR TITLE
Sexyplate

### PR DESCRIPTION
**Pull request by akhamar**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folder and file naming:
    * Folders and filenames shouldn't contain spaces. Only letters `a-zA-Z`, numbers `0-9`, underscores `_`, hyphens `-` and periods `.`
    * Primary color: `part_xyz.stl`
    * Accent color: `[a]_part_xyz.stl`
    * Opaque color (Blocks light): `[o]_part_xyz.stl`
    * Clear/transparent color (Allows light): `[c]_part_xyz.stl`
    * Quantity, if more than one is needed: `part_xyz_x4.stl`
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

A mod that allow to store steel sheet plate on the side of the printer, next to the panels.

#### How was it tested?

On my voron 2.4

#### Any background context you want to provide?

None found, created one.

#### Screenshots (if appropriate)

CF images folder.

### CODE REVIEW

**Review by Trippinonaduck — COMMENTED** [review]
This should also work on a Trident and 1.8.  I recommended foam size would also be useful to fit the size you exported stls for.

**Comment by akhamar** [comment]
> This should also work on a Trident and 1.8. A recommended foam size would also be useful to fit the size you exported stls for.

Acrylic panels vary in thickness, and people can use caliper or ruler if then need to know.

**Comment by Trippinonaduck** [comment]
> 

Sorry, let me clarify. Not everyone will have the ability to measure your stl or export their own custom size. What size did you export them at so that people will know what foam/panel thickness the currently provided stls will accommodate? That information would be helpful to include in the readme.

**Comment by akhamar** [comment]
> > 
> 
> Sorry, let me clarify. Not everyone will have the ability to measure your stl or export their own custom size. What size did you export them at so that people will know what foam/panel thickness the currently provided stls will accommodate? That information would be helpful to include in the readme.

Sorry my bad I missunderstood with my other mod (sexyzip).
I'll export 3 version 4mm, 5mm and 6mm. The current one is 5mm.

**Comment by akhamar** [comment]
Done

File: printer_mods/README.md [hunk 1]
```diff
@@ -268,6 +268,7 @@ like so:
 |madros |[V0 OLED-skirt](./madros/V0_skirt_OLED) |V0 skirt with 0.96" OLED|:heavy_check_mark: |:x: |:x: |:x:|
 |MarcPot |[UHP PSU Bracket Clip](./MarcPot/UHP_PSU_Bracket_Clip)|DIN rail mount for the Mean Well UHP series PSU. |:grey_question: |:grey_question: |:heavy_check_mark: |:x:|
 | |[Hour Counter Skirt + Keystone Skirt](./MarcPot/Skirt_Mods)|Skirt mod for a mechanical hour counter and keystone inserts | :x: |:x: |:heavy_check_mark: |:x:|
+|Mastah |[SexyPlate](./Mastah/sexyplate/)|Steel sheet plate holder on the side of the printer |:grey_question: |:grey_question: |:heavy_check_mark: |:grey_question: | :grey_question: |
```
Comment (MapleLeafMakers) [inline printer_mods/README.md:271]: I don't see any reason why this wouldn't work on a Trident or 1.8 as well, those can probably get the ✔️

Comment (akhamar) [reply printer_mods/README.md:271]: Can't test so not really whiling to put check mark there.
If people test and actually give a go for those printer, happy to add them in readme.

**Comment by FHeilmann** [comment]
Hi, 

I want to thank you for your submission and want to personally apologize that your submission has not gotten the review it deserves.
We are in the process to restructure the repo to streamline reviews and hopefully solve this issue in the future.

Unfortunately, this means that we will require you to resubmit the mod under the new system.

Please check [here](https://github.com/VoronDesign/VoronUsers/wiki/VoronUsers-Overhaul) for details and instructions on how to proceed. 
If you have any questions, please reach out in the #voronuser_mods channel on the VoronDesign discord.
If you have since moved on from this mod and don't want to resubmit, no further action is necessary from your side.

Bests,

Florian a.k.a. Haribro

**Pull request closed without merging**